agent/installer/steps.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package installer
+
+// contains the installation and uninstallation steps for the supported os and k8s
+var (
+	DoUbuntu20_4K8s1_22 = `
+set -euo pipefail
```

Comment:
Do we also need to set `#!/usr/bin/env bash` at the top of the script as it will run as a bash script?

Reply:
we will be executing this script using `cmd := exec.Command("/bin/bash", DoUbuntu20_4K8s1_22)` so `#!/usr/bin/env bash` won't be  required here.